### PR TITLE
libretro: Add Travis testing

### PR DESCRIPTION
**Pull request by RobLoach**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,78 +1,36 @@
...
+    sources:
+      - ubuntu-toolchain-r-test
+env:
+  global:
+    - CORE=citra
+    - COMPILER_NAME=gcc CXX=g++-7 CC=gcc-7
+  matrix:
+    - PLATFORM=3ds
+    - PLATFORM=linux_x64
+    - PLATFORM=ngc
```
Comment (j-selby) [inline .travis.yml]: Not sure what these other platforms are here? Don't think we are running on the GameCube anytime soon :P

Comment (RobLoach) [reply .travis.yml]: Agreed. Looks like those platforms pass, but it's a false positive. Removed them.

**Comment by RobLoach** [comment]
@bparker06 did magic to make it use the currently checked out code:
https://github.com/libretro/libretro-super/blob/master/travis/build-linux_x64.sh

**Comment by j-selby** [comment]
Thanks!

**Pull request merged**